framework/packages/abstract-client/src/account.rs

```diff
@@ -176,17 +207,30 @@ impl<Chain: CwEnv> Account<Chain> {
     }
 }
 
-fn get_sub_account_id_from_events(events: Vec<Event>) -> Option<u32> {
+struct ExtractedEventInfo {
```

Comment:
What "extracted info"? This type could use a better name

Reply:
Haha yeah, I am not too sure how else to call it. The two things we extract are not really related as far as I can tell so this was the best I could come up with. Any ideas?

Reply:
Ok, how is `CreationAttributeValues`? Since all of the values are things that were "created".

Reply:
`ParsedAccountCreationEvent`?

Reply:
How about `ParsedAccountCreationResponse`? It made the related function easier to name as well.

Reply:
Sure!